setup.py

```diff
@@ -40,6 +40,7 @@ def get_long_description():
     install_requires=[
         "anyio>=3.0.0,<4",
         "typing_extensions; python_version < '3.8'",
+        "contextlib2 >= 21.6.0; python_version < '3.7'",
```

Comment:
Out of interest, why the `21.6.0`? I see some typing fixes in the repo but not sure what applies here?

Reply:
It adds asynccontextmanager